recipes/eip712/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "eip712" %}
+{% set version = "0.1.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/eip712-{{ version }}.tar.gz
+  sha256: 478ffd839daeb921d508055e063315080aaa112ec8aa1621ef858a7f249469cc
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8,<4.0
+    - setuptools-scm
+    - pip
+  run:
+    - python >=3.8,<4.0
+    - dataclassy >=0.8.2,<1
+    - eth-utils >=2.1.0,<3
+    - eth-abi >=3.0.1,<4
+    - eth-typing >=2.3.0,<4
+    - hexbytes >=0.3.0,<1
+    - pycryptodome >=3.16.0,<4
+
+test:
+  imports:
+    - eip712
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/ApeWorX/eip712
+  summary: 'eip712: Message classes for typed structured data hashing and signing in Ethereum'
+  license: Apache-2.0
+  license_file: license.txt
+
+extra:
+  recipe-maintainers:
+    - spinoch
```

Comment:
In general this should include your username since you're submitting the recipe and would be expected to maintain the recipe within conda-forge. You are welcome to include others, but we need confirmation from them that they are willing to be listed as maintainers for the recipe.

Reply:
thank you, I have previous confirmation from them
@spinoch can you please confirm here oficially that you want to be the maintainer?